src/gsSolver/gsMatrixOp.h

```diff
@@ -49,42 +49,40 @@ class gsMatrixOp : public gsLinearOperator<typename MatrixType::Scalar>
     /// @note This does not copy the matrix. Make sure that the matrix
     /// is not deleted too early (alternatively use constructor by
     /// shared pointer)
-    gsMatrixOp(const MatrixType& mat, bool sym=false)
-    : m_mat(), m_expr(mat.derived()), m_symmetric(sym)
+    gsMatrixOp(const MatrixType& mat)
+    : m_mat(), m_expr(mat.derived())
     {
         //gsDebug<<typeid(m_expr).name()<<" Ref: "<<is_ref<NestedMatrix>::value<<"\n";
     }
 
     /// @brief Constructor taking a shared pointer
-    gsMatrixOp(const MatrixPtr& mat, bool sym=false)
-    : m_mat(mat), m_expr(m_mat->derived()), m_symmetric(sym)
+    gsMatrixOp(const MatrixPtr& mat)
+    : m_mat(mat), m_expr(m_mat->derived())
```

Comment:
I think that
gsMatrixOp(MatrixPtr mat) : m_mat(give(mat))

is more suitable here

Reply:
done